packages/jbrowse-plugin-apollo/src/components/AddFeature.tsx

```diff
@@ -110,7 +110,7 @@ export function AddFeature({
       },
       parentFeatureId: sourceFeature._id,
     })
-    changeManager.submit?.(change)
+    await changeManager.submit?.(change)
```

Comment:
oh man, I would have hoped we had a lint rule that would have caught that

Reply:
@typescript-eslint/no-floating-promises may help catch some things like this, it is part of @typescript-eslint/recommended-requiring-type-checking (which needs to be set separately from @typescript-eslint/recommended)

Reply:
@kyostiebi Can you confirm that this change is necessary to fix the issue? I don't see how this would change anything on the backend.

Reply:
I just tested that I submitted three times _AddFeatureChange_ -change from UI. Calling _changeManager.submit?.(change)_ without _await_, I got the following error:
[_Nest] 6320  - 06/05/2023, 3:04:46 PM   DEBUG [ParseChangePipe] *** FAILED: MongoServerError: WriteConflict
[Nest] 6320  - 06/05/2023, 3:04:46 PM   DEBUG [ChangesService] *** INSERT DATA EXCEPTION - Start to clean up old temporary documents...
[Nest] 6320  - 06/05/2023, 3:04:46 PM   ERROR [GlobalExceptionsFilter] UnprocessableEntityException: MongoServerError: WriteConflict
[Nest] 6320  - 06/05/2023, 3:04:46 PM   ERROR [GlobalExceptionsFilter] UnprocessableEntityException: MongoServerError: WriteConflict_

With _await_ keyword, all three changes were processed fine.